portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase

```diff
@@ -0,0 +1,235 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		HeadlessSite.addSite(siteName = "Test Site Name");
+
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Blogs Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			column = 1,
+			groupName = "Test Site Name",
+			layoutName = "Blogs Page",
+			widgetName = "Blogs");
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Test Site Name");
+	}
+
+	tearDown {
+		PortalInstances.tearDownCP();
+
+		JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+	}
+
+	@description = "Can Create a Blog"
+	@priority = 5
+	test CanCreateBlog {
+		task ("When the user adds a Blog Entry") {
+			BlogsNavigator.openToAddEntry(
+				layoutName = "Blogs Page",
+				siteURLKey = "test-site-name");
+
+			BlogsEntry.addEntryContent(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title 1");
+
+			Button.clickPublish();
+		}
+
+		task ("Then the Blog Entry should display on the page") {
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Test Site Name");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title 1");
+		}
+	}
+
+	@description = "Can delete Blog"
+	@priority = 5
+	test CanDeletBlog {
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "test-site-name");
+
+		BlogsEntry.addEntryContent(
+			entryContent = "My Content",
+			entryTitle = "My Blogs");
+
+		Button.clickPublish();
+
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "test-site-name");
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Test Site Name");
+
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "test-site-name");
+
+		BlogsEntry.addEntryContent(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry Title");
+
+		Button.clickPublish();
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Test Site Name");
```

Comment:
I think from line 63 to 93 should be replaced by a simple json macro to add a second blog entry since it's not the main goal of this test.. 
Don't forget adding a task for this step.